epub33/rs/index.html

```diff
@@ -838,6 +839,10 @@ <h3>Scripting</h3>
 					</li>
 				</ul>
 
+				<p class="ednote">Note that the definition of <code>epubReadingSystem</code> is currently markes as "at
+					risk". If, in the final version of this document, the object becomes non-normative, then each "MUST"
```

Comment:
This is not a comment on this line, but I guess I can't comment on unchanged lines? This interface is weird.

Anyway, line 833 above has JavaScript navigator. Should we drop "JavaScript", since you dropped it below?

Reply:
I would think we should link/reference the definition in the HTML spec.

Reply:
I've removed the references to javascript and the navigator object from the reading system requirement as this is explained in the object definition. I've also added a link on the navigator object reference where it's introduced in the definition.